src/lib/merkle_ledger_tests/test_mask.ml

```diff
@@ -188,17 +188,6 @@ module Make (Test : Test_intf) = struct
         (* verify all hashes to root are same in mask and parent *)
         compare_maskable_mask_hashes maskable attached_mask dummy_address )
 
-  let%test "mask delegates to parent" =
```

Comment:
> Remove invalid test: mask parents should not be changed

Is this actually an invalid test ? Masks parents are sometimes changed/modified.
It is the reason why there is the method `Mask.Attached.parent_set_notify`:
https://github.com/MinaProtocol/mina/blob/552852851deb23a697c8078c223a3181eb225361/src/lib/merkle_mask/maskable_merkle_tree.ml#L227

Removing this test means we're breaking the relation parent/child described in [specs/merkle_tree.md](https://github.com/MinaProtocol/mina/blob/develop/docs/specs/merkle_tree.md)

Reply:
@georgeee Can we add this test back? If the dummy account is added before the child mask is attached, it should do as expected, right?